ld.so.conf

```diff
@@ -0,0 +1,3 @@
+include /run/flatpak/ld.so.conf.d/*.conf
+include /app/etc/ld.so.conf
+/app/lib
```

Comment:
Do we need this? I mean, any flatpak that supports ld.so.cache will generate an identical one automatically.

Reply:
Yes. I think needed that for some testing.

Reply:
We do not need it.